app/main.py

```diff
@@ -1,3 +1,25 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        full_weeks = 0
+        if days <= 7:
+            return 1
+        else:
+            full_weeks = days / 7
+            return math.ceil(full_weeks)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        days = course_dict['days']
+        weeks = cls.days_to_weeks(days)
+        the_instance = OnlineCourse(course_dict['name'],
```

Comment:
and here you should write `cls` instead of `OnlineCourse`. If you inherited this class, your method would not work correctly in child classes